src/org/jitsi/impl/neomedia/codec/FMJPlugInConfiguration.java

```diff
@@ -109,8 +109,8 @@
             //"org.jitsi.impl.neomedia.codec.video.h263p.Packetizer",
             // Adaptive Multi-Rate Wideband (AMR-WB)
             // "org.jitsi.impl.neomedia.codec.audio.amrwb.DePacketizer",
-            //"org.jitsi.impl.neomedia.codec.audio.amrwb.JNIDecoder",
-            //"org.jitsi.impl.neomedia.codec.audio.amrwb.JNIEncoder",
+            "org.jitsi.impl.neomedia.codec.audio.amrwb.JNIDecoder",
+            "org.jitsi.impl.neomedia.codec.audio.amrwb.JNIEncoder",
```

Comment:
I think this should be added based on the ffmpeg check you do in MediaUtils? WDYT? Will that work?

Reply:
Mhm. That is done in the second commit above. I worked with the Web interface of GitHub and did not get a button to squash those commits (why ever). Was that your question? So, the commit squash is missing.

Reply:
Nope, I do not see this change. Look at the CUSTOM_CODECS array how g729 encoder&decoder are skipped.
I'm just not sure whether FMJPlugInConfiguration or MediaUtils is loaded first from JVM so that can be done with a public static field ...

Reply:
I do not understand what you mean. This has nothing to do with G.729. This Pull Request has two commits. The second commit `1cceccb` re-enables AMR-WB. Because of the first commit `82ecd62`, AMR-WB gets added conditionally if FFmpeg offers it.

Reply:
I'm talking just for the second one. Encoder and decoder to be conditionally added only if amr is offered from FFmpeg. 
G729 was just an example, where the encoder and decoder are conditionally added.

Reply:
I have no idea what you are talking about. If you provide a code snippet outlining your request/idea (?), that would help. AMR-WB follows the pattern of H.264. If you want to have something added here, I recommend to pass this Pull Request and then change those media formats with a unique change. That way, those two changes are isolated and can be tracker more easily.

G.729 is conditionally disabled but that is actually not a runtime- but a compile-time flag. AMR-WB (and H.264) can be detected at runtime. For example in Linux, I change from libavcodec-extra to libavcodec. On the next restart of Jitsi Desktop, AMR-WB is disabled. Tested. For example in Linux, I remove libavcodec (or the whole FFmpeg) completely, then both AMR-WB and H.264 are removed. Tested. I did not recompile or make anything; this is runtime. Because LibJitsi includes just wrappers for AMR-WB (in contrast to G.729 where it is a complete own implementation), I do not see any patent/license issue. That is why I do not understand your reference to G.729.

Reply:
> For example in Linux, I change from libavcodec-extra to libavcodec. On the next restart of Jitsi Desktop, AMR-WB is disabled.

In this case do you see a stack trace from AMR Encoder/Decoder trying to load?

Reply:
Yes, as with H.264. Therefore, you are asking for an orthogonal feature.

Reply:
I would argue that. I need to test it, but I remember I was trying to surpress printing those. And normally you will have h264, while amr you will not have and those will fill the logs on every run. 
This is a change enabling it, we need to take care of those prints at the same time.

Reply:
I will check this next week. I tried updating Jitsi to latest today, but it needed more work than just replacing few jars ...

Reply:
I do not have H.264, if I want to – and Jitsi still works. If I change that here for AMR, I have to change it for H.264 as well. Therefore, this is an orthogonal issue.

By the way, what do you mean with stack trace? Where shall that be printed? I started Jitsi Desktop from the console and got nothing. I inserted a `System.err.println` to see whether those classes get loaded. Or are you about a certain log (level)?